### ISSUE TITLE
Problems using MEQP2 code sniffer rule set

### ISSUE THREAD

**Issue by siment** [squizlabs/PHP_CodeSniffer#1416]
Hi,

and thanks for implementing the Magento 2 sniffer rule set from the Magento Marketplace Extension Quality Program.

However, it does not seem to work. The error that is reported on the builds in Code Climate is the following:

`PHP Fatal error:  Cannot declare class MEQP2\Sniffs\Classes\ObjectInstantiationSniff, because the name is already in use in /usr/src/app/vendor/magento/marketplace-eqp/MEQP2/Sniffs/Classes/ObjectInstantiationSniff.php on line 15` 

I am using a `phpcs.xml.dist` file to configure the sniffer - see below:

```xml
<?xml version="1.0"?>
<ruleset name="Module_Name">
  <description>The coding standard for Module_Name</description>
  <file>src/</file>
  <file>Test/</file>
  <arg name="extensions" value="php"/>
  <rule ref="MEQP2" />
</ruleset>
```

I have also tried relative path to rule set like so:

```xml
<?xml version="1.0"?>
<ruleset name="Module_Name">
  <!-- SNIP -->
  <rule ref="relative/path/to/MEQP2/ruleset.xml" />
</ruleset>
```

Thanks for looking into this.

**Comment by gsherwood** [comment]
I think you've posted this in the wrong project. The PHP_CodeSniffer project does not contain a standard for Magento. You must have got this standard from somewhere else, so you'll want to ask them what the best way to install and use it is.

**Comment by siment** [comment]
Ugh, I must have had multiple tabs opened and accidentally posted on the wrong project. Delete the issue completely if you can. Sorry about that.

**Comment by gsherwood** [comment]
> Ugh, I must have had multiple tabs opened and accidentally posted on the wrong project.

No worries. Thanks for getting back to me.
